.github/workflows/cmfgen_data_check.yml

```diff
@@ -0,0 +1,69 @@
+name: Check CMFGEN data update workflow
+
+on:
+  schedule:
+    - cron: '0 0 * * 5' 
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
+
+    steps:
+      - uses: act10ns/slack@v2.0.0
+        with:
+          status: starting
+          message: Starting Check CMFGEN data update workflow
+        if: always()
+        
+      - name: Checkout github repo
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+
+      - name: Configure git
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@github.com"
+                
+      - name: Make temporary directories
+        id: setup
+        run: |
+          mkdir ~/temp
+          
+      - name: Download data
+        id: download_data
+        run: |
+          SOURCE_URL=https://kookaburra.phyast.pitt.edu/hillier/cmfgen_files/atomic_data_15nov16.tar.gz
+          wget -q -U "Mozilla/4.0" $SOURCE_URL -O ~/temp/atomic.tar.gz
+          tar -zxf ~/temp/atomic.tar.gz 
+          
+      - name: Commit files to git
+        id: commit_changes
+        run: |
+          git add .
+          git diff --quiet --exit-code --cached
+          exit_code=$?
+          if [ $exit_code -eq 0 ]; then
+            echo "No changes in the CMFGEN data to commit"
+            exit 0
+          else
+            git commit -m "New dataset added"
```

Comment:
We should look into either a separate job that mentions the data was updated or/and this should be reflected on slack. A job  like this perhaps?

https://github.com/tardis-sn/carsus/blob/13bda77af25e1f375dbba9040c87a813e47e64f7/.github/workflows/save_atomic_files.yml#L38

Should be fairly easy to do. Let me know if you need help

Reply:
I have made some changes, you can chcek.